repository/src/main/resources/alfresco/repository.properties

```diff
@@ -1012,6 +1012,10 @@ local.transform.service.enabled=true
 local.transform.service.cronExpression=4 30 0/1 * * ?
 local.transform.service.initialAndOnError.cronExpression=0 * * * * ?
 
```

Comment:
Change comment: A switch to enable or disable the use of Direct Access URLs in transforms. Must also be supported for the source node.